staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -114,28 +114,12 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope *RequestSc
 			return
 		}
 
-		trace.Step("About to check admission control")
-		if admit != nil && admit.Handles(admission.Delete) {
```

Comment:
before we merge this, we need a test that exercises delete and deletecollection on all the resources and subresources that support those verbs (including custom resources) and ensures admission is called.https://github.com/kubernetes/kubernetes/pull/76677 is starting a test like that to exercise webhook admission... can wait for that or do something simpler with non-webhook admission as part of this PR.